interfaces/Portal/src/assets/i18n/en.json

```diff
@@ -257,7 +257,7 @@
         "titlePortal": "Title portal"
       },
       "program-payout": {
-        "all-closed-payments": "All closed payments",
+        "all-closed-payments": "Last 5 closed payments",
```

Comment:
I think this key also needs an update `all-closed-payments` to something like `last-closed-payments` or something along those lines

Could you also do the dutch translation in the nl.json: 'De laatste 5 gesloten betalingen'

Reply:
Ah also maybe the 5 could be dynamic in the string?